main.js

```diff
@@ -87,16 +86,31 @@ function saveOutfit() {
   localStorage.setItem('id', ++curId);
   currentOutfit.title = titleInput.value;
   outfits.push(currentOutfit);
-  createSavedOutfitCard();
+  createSavedOutfitCard(currentOutfit);
   clearBearDisplay();
   titleInput.value = '';
   enableSaveButton();
+  addOufitsToLocalStorage()
   currentOutfit = new Outfit();
 }
 
-function createSavedOutfitCard() {
+function addOufitsToLocalStorage() {
```

Comment:
got a typo, should be named addOutfitsToLocalStorage, you're missing the first 'T' in outfits

Reply:
Nice catch